docs/modules/operation/pages/deep-dive/flows/sentinel/message-broker/activemq.adoc

```diff
@@ -30,9 +30,10 @@ config:property-set broker-url failover:tcp://my-activemq-ip:61616<4>
 config:update
 ----
 
-<1> A location string is used to assign the Sentinel to a monitoring location with Minions.
-<2> Unique identifier used as a node label for monitoring the Sentinel instance within {page-component-title).
-<3> URL that points to ActiveMQ broker.
+<1> A location string is used to assign the Sentinel to a monitoring location in your environment with Minions.
```

Comment:
```suggestion
<1> A location string is used to assign the Sentinel to a monitoring location in your environment that has Minions.
```
Please ignore if this is not the meaning. I wasn't clear on the exact intent here.